dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/SignIn/Trn/OfficialNameTests.cs

```diff
@@ -0,0 +1,134 @@
+namespace TeacherIdentity.AuthServer.Tests.EndpointTests.SignIn.Trn;
```

Comment:
No tests for showing/hiding conditional block on radio button select, not sure if necessary?

Reply:
GDS library covers that so I don't think we need it.